packages/core/src/routes/logto-config.ts

```diff
@@ -34,4 +45,108 @@ export default function logtoConfigRoutes<T extends AuthedRouter>(
       return next();
     }
   );
+
+  router.get(
+    '/configs/oidc-keys',
+    koaGuard({
+      response: getOidcKeysResponseGuard,
+      status: [200, 404],
+    }),
+    async (ctx, next) => {
+      const configs = await getOidcConfigs();
+
+      // Remove actual values of the private key from response
+      const resultWithoutPrivateKey = Object.fromEntries(
+        Object.entries(configs).map(([key, value]) => [
+          key,
+          value.map(({ id, createdAt }) => ({ id, createdAt })),
+        ])
+      );
+
+      ctx.body = resultWithoutPrivateKey;
+
+      return next();
+    }
+  );
+
+  router.delete(
+    '/configs/:configKey/:keyId',
+    koaGuard({
+      params: z.object({
+        configKey: z.nativeEnum(LogtoOidcConfigKey),
+        keyId: z.string(),
+      }),
+      status: [204, 404, 422],
+    }),
+    async (ctx, next) => {
+      const { configKey, keyId } = ctx.guard.params;
+      const { rows } = await getRowsByKeys([configKey]);
+
+      const result = z.array(oidcPrivateKeyGuard).safeParse(rows[0]?.value);
+
+      if (!result.success) {
+        throw new RequestError({ code: 'request.general', status: 422 });
+      }
+
+      if (result.data.length <= 1) {
+        throw new RequestError({ code: 'oidc.key_required', status: 422 });
+      }
+
+      if (!result.data.some(({ id }) => id === keyId)) {
+        throw new RequestError({ code: 'oidc.key_not_found', id: keyId, status: 404 });
+      }
+
+      const updatedConfigValue = result.data.filter(({ id }) => id !== keyId);
+
+      await updateOidcConfigsByKey(configKey, updatedConfigValue);
+      await envSet.load();
+
+      ctx.status = 204;
+
+      return next();
+    }
+  );
+
+  router.post(
```

Comment:
Want to double check the endpoints in this route is only available to admin users.

Reply:
Yes